libvast/vast/format/json/suricata.hpp

```diff
@@ -0,0 +1,102 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#pragma once
+
+#include <unordered_map>
+
+#include "vast/concept/printable/vast/json.hpp"
+#include "vast/detail/line_range.hpp"
+#include "vast/detail/overload.hpp"
+#include "vast/error.hpp"
+#include "vast/expected.hpp"
+#include "vast/format/multi_layout_reader.hpp"
+#include "vast/json.hpp"
+#include "vast/schema.hpp"
+#include "vast/table_slice_builder.hpp"
+#include "vast/view.hpp"
+
+namespace vast::format::json {
+
+struct suricata {
+  caf::optional<vast::record_type> operator()(const vast::json::object& j) {
+    auto path = j.find("event_type");
```

Comment:
Why is this variable called `path`? I would find simple iterator naming more intuitive.

```suggestion
    auto i = j.find("event_type");
```

(My hunch is that this is from the Zeek streaming JSON, which uses a `_path` field.)

Reply:
Your guess is correct. It was a copy & paste.